.github/workflows/deploy-storybook.yml

```diff
@@ -28,8 +28,8 @@ jobs:
       - name: Install Dependencies
         run: rush install
 
-      - name: Build Storybook dependencies
-        run: rush build -v -T storybook
+      - name: Build Storybook
+        run: rush build -v -o storybook
```

Comment:
What does the `-o` flag do here?

Reply:
https://rushjs.io/pages/commands/rush_build/
short for "--only"
telling rush to only build storybook